sdk/core/Azure.Core/src/Response.cs

```diff
@@ -86,9 +86,9 @@ public virtual BinaryData Content
         /// Indicates whether the message's <see cref="ResponseClassifier"/> considers this
         /// response an error.
         /// </summary>
-        public bool IsError { get; internal set; }
+        public virtual bool IsError { get; internal set; }
```

Comment:
Does Moq work when the setter is internal?

Reply:
Yes - adding a test for this.